### ISSUE TITLE
package claims python3.8 compatibility, but doesn't work

### ISSUE THREAD

**Issue by fixthething** [mwiens91/english-words-py#6]
Some of the type annotation is python 3.9+ compatible but causes an error on python3.8. ("set" vs "typing.Set"?). Could change the project metadata to remove python3.8 support if you want to lean into the future (python 3.9 typing is the way it should have been from the start, IMHO), or else change the typing.

`$ python3 -m pip install english_words --upgrade
Collecting english_words
  Downloading english-words-2.0.0.tar.gz (8.2 MB)
     |████████████████████████████████| 8.2 MB 5.3 MB/s
  Installing build dependencies ... done
  Getting requirements to build wheel ... error
  ERROR: Command errored out with exit status 1:
   command: /usr/bin/python3 /tmp/tmp3a_bf23s get_requires_for_build_wheel /tmp/tmp2v3fwjm1
       cwd: /tmp/pip-install-bjppp9b3/english-words
  Complete output (19 lines):
  Traceback (most recent call last):
    File "/tmp/tmp3a_bf23s", line 280, in <module>
      main()
    File "/tmp/tmp3a_bf23s", line 263, in main
      json_out['return_val'] = hook(**hook_input['kwargs'])
    File "/tmp/tmp3a_bf23s", line 114, in get_requires_for_build_wheel
      return hook(config_settings)
    File "/tmp/pip-build-env-72rz24cn/overlay/lib/python3.8/site-packages/setuptools/build_meta.py", line 341, in get_requires_for_build_wheel
      return self._get_build_requires(config_settings, requirements=['wheel'])
    File "/tmp/pip-build-env-72rz24cn/overlay/lib/python3.8/site-packages/setuptools/build_meta.py", line 323, in _get_build_requires
      self.run_setup()
    File "/tmp/pip-build-env-72rz24cn/overlay/lib/python3.8/site-packages/setuptools/build_meta.py", line 487, in run_setup
      super(_BuildMetaLegacyBackend,
    File "/tmp/pip-build-env-72rz24cn/overlay/lib/python3.8/site-packages/setuptools/build_meta.py", line 338, in run_setup
      exec(code, locals())
    File "<string>", line 4, in <module>
    File "/tmp/pip-install-bjppp9b3/english-words/english_words/__init__.py", line 16, in <module>
      ) -> set[str]:
  TypeError: 'type' object is not subscriptable
  ----------------------------------------
ERROR: Command errored out with exit status 1: /usr/bin/python3 /tmp/tmp3a_bf23s get_requires_for_build_wheel /tmp/tmp2v3fwjm1 Check the logs for full command output.`

**Comment by guy4261** [comment]
need to change

```diff
+from typing import Set
...
-) -> set[str]:
+) -> Set[str]:

```

**Comment by guy4261** [comment]
PR: #8

**Comment by mwiens91** [comment]
Sorry for taking so long to get to this. I have removed Python 3.8 support in the metadata in 8f6ee885860fcb46f96eecd87226517d685b4ee4. guy4261's fix in #8 will work for Python 3.8, but have decided not to merge it in favour of using the newer 3.9+ notation.
